src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -2997,7 +2999,7 @@ public object VisitPipeline(PipelineAst pipelineAst)
                 Expression input;
                 int i, commandsInPipe;
 
-                if (firstCommandExpr != null)
+                if (firstCommandExpr != null  && !pipelineAst.BackgroundProcess)
                 {
```

Comment:
I see no tests to cover this case - are you sure it's correct?

A test would look like:

```powershell
1..10 | % { $_ } &
```